packages/react-art/src/ReactARTHostConfig.js

```diff
@@ -445,7 +445,9 @@ export function handleEventComponent(
 
 export function handleEventTarget(
   type: Symbol | number,
-  props: Props,
+  lastProps: Props,
```

Comment:
I think we should keep the name as `props` if these are the currently-in-use props (rather than the previous props)

Reply:
We use lastProps, nextProps naming through all the host configs.